doc/rst/technotes/gpu.rst

```diff
@@ -0,0 +1,90 @@
+.. _readme-gpu:
+
+.. default-domain:: chpl
+
+=============================
+Support for GPU targeted code
+=============================
+
+Chapel includes preliminary work to target Nvidia GPUs using CUDA. This work is
+under active development and has not yet been tested under a wide variety of
+environments. We have tested it on systems with Nvidia Tesla P100 GPUs and CUDA
+11.0 and a system with Nvidia Tesla K20 GPUs with CUDA 10.2. The current
+implementation does not support multiple GPUs on one node and will only apply
+to certain `forall` loops.
```

Comment:
I think the correct markup for inline code is double backtics. ` ``forall`` `

Reply:
I see. I guess the `.rst` files are restructured text and the `.MD` files are markdown (which is what's used in Github comments)? And I guess you use a single backtick in markdown and double in rst?

Anyway I've changed them all to use double backticks.